web/beacon-app/package.json

```diff
@@ -30,6 +30,8 @@
     "watch:css": "npx tailwindcss -o ./public/tailwind.css --watch",
     "precommit": "lint-staged",
     "cypress": "cypress open --e2e --browser chrome",
+    "cypress:report": "cypress run --e2e --browser chrome --reporter mochawesome --reporter-options reportDir=cypress/reports,overwrite=false,html=false,json=true",
```

Comment:
Should the code coverage appear in a browser after running the `yarn cypress:report` command?

Reply:
we could see the report in the browser while testing and also have it in this folder `reportDir=cypress/reports`